addons/pos_stripe/static/src/js/payment_stripe.js

```diff
@@ -125,16 +125,18 @@ let PaymentStripe = PaymentInterface.extend({
 
         const intentCharge = charges.data[0];
         const processPaymentDetails = intentCharge.payment_method_details;
-        const cardPresentBrand = processPaymentDetails.card_present.brand;
-
-        if (processPaymentDetails.type === 'interac_present') {
-            // Canadian interac payments should not be captured:
-            // https://stripe.com/docs/terminal/payments/regional?integration-country=CA#create-a-paymentintent
-            return ['interac', intentCharge.id];
-        } else if (cardPresentBrand.includes('eftpos')) {
-            // Australian eftpos should not be captured:
-            // https://stripe.com/docs/terminal/payments/regional?integration-country=AU
-            return [cardPresentBrand, intentCharge.id];
+        const cardPresent = processPaymentDetails.card_present;
+        if (cardPresent) {
```

Comment:
could you replace this with
```js
if( !cardPresent ) {
        return [false, false];
}
```
This way you don't have all the extra diff and will also have less indentation